Tools/scripts/deepfreeze.py

```diff
@@ -86,12 +87,17 @@ def analyze_character_width(s: str) -> tuple[int, bool]:
     return kind, ascii
 
 
+def removesuffix(base, suffix):
```

Comment:
```suggestion
def removesuffix(base: str, suffix: str) -> str:
```

Since other functions are annotated so annotate this too ?